### ISSUE TITLE
[Feature Request] A block (machine) to resort contents of storage cells OR (Export+Storage) Bus.

### ISSUE THREAD

**Issue by FREEZEball** [bdew/ae2stuff#50]
Normally you would do it via an [ME I/O Port](http://ae-mod.info/ME-IO-Port/): put a cell into it to resort cells' contents. You need it, if you organize a new [container](http://ae-mod.info/ME-Storage-Bus/) for stuff that is already in the network. ME system only exports newly coming items into a storage cell.

OR add **sort of** a ME Storage Bus, that also has functions of [ME Export Bus](http://ae-mod.info/ME-Export-Bus/) and exports items into container with high priority. OR maybe an upgrade card for existing ME Storage Buses, that allows them to put items from network into connected storage.

Sorry about my poor English :)

**Comment by bdew** [comment]
This is an interesting idea, thogh i have no idea how to implement it. I might look into it in the future.

**Comment by xBlizzDevious** [comment]
I second this - I tried moving from Refined Storage + Storage Drawers (with a storage bus) to an AE2 system with fully upgraded YABBA barrels. It took a VERY long time to transfer the millions of items I had stored with export buses and item conduits!

A block that I can place and tell it to move all items from the network into another block or vice versa would be incredibly useful.

EDIT: Just noticed this is nearly two years old...
